config/config.go

```diff
@@ -358,6 +358,16 @@ func InitConfig() {
 		}
 		SetLogging(level)
 	}
+
+	logLocation := param.Logging_File.GetString()
+	if logLocation != "" {
+		// Note: do not need to close the file, logrus does it for us
+		f, err := os.OpenFile(logLocation, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+		if err != nil {
```

Comment:
I don't think you need this `err != nil` line as cobra.CheckErr does that check already. It's redundant.

Reply:
Ah, good catch